_amenities/pools/pool-rules.md

```diff
@@ -0,0 +1,82 @@
+
+* Infants/children not toilet trained must wear swim diapers under swim suits in order to prevent contamination.
+* Children must have their diapers changed on changing tables in the locker rooms.
+* Everyone must take a cleansing shower before using the pool.
+* All accidents should be reported to the lifesaving personnel immediately.  A first aid kit is available in the guard office.
+* In case of emergency, a 9-1-1 phone is located next to the boy's locker room door facing the pool deck.
+* NO RUNNING on the pool deck or in restrooms.
+* NO HORSEPLAY or ROUGH-HOUSING.
+* Do not hang on the lane lines or the rope.
+* Do not open skimmers.
+* Starting blocks may be used only in scheduled practices, competitions, and instruction when supervised by a coach or instructor.
+* Kickboards are for lap swimming, swim team, and swim lessons ONLY.
+* Jumps into the pool must be feet first and facing the pool.  NO SPINS, FLIPS, OR DIVES.
+* Inflatable beach balls or sponge balls are allowed during pool non-crowdedconditions and at the manager's discretion.
+* No smoking inside the pool area.
+
+The Aquatics Manager or Lead Lifeguard may close the pool complex, or any portions of the pool complex for any safety or maintenance reasons.
+
+Animals are not allowed in the pool area, with the exception of service dogs.  Service dogs are allowed on deck, but not in the water.
+
+Food and beverages are permitted on the pool deck, but are not allowed to be consumed while in the pool.
+
+The follow items are not allowed in the pool:
+* No water guns
+* No hard balls (such as tennis balls, footballs,etc.)
+* No flotation devices (inner tubes, inflatablerafts, noodles, etc.)
+* No kickboards or buoys during open swim times
+* No playing, climbing, or jumping off of the metal ladders or hand rails.
+* No sitting on  or riding on the shoulders of other patrons.
+
+#### Prohibited acts/items
+* Running, unnecessary noise, boisterous/roughplay
+* Shoulder-riding  /  Tennis balls  /  Squirt guns
+* Animals of all kinds  /  Smoking
+* Food or drink in not allowed in the water
+* Glass containers  /  Alcoholic beverages  /  Diving
+* Rafts or large flotation devices  /  Inappropriate swimming attire
+* Disposable diapers
+
+#### General Pool Rules
+Pool use is reserved for Klahanie Association members in good standing and their guests only, unless otherwise scheduled.
+
+* All children 12 years old and under must be supervised and accompanied by a parent or guardian.
+* Patrons ages 13 - 17 years old must not use the pool alone and must be accompanied by a buddy.
+* Each Klahanie household may sponsor a maximum of 5 guests on any given day and meet the requirements of item above.

```

Comment:
Once we have a style guide, we need to revisit the use of numbers throughout the site. Chicago Manual of Style recommends spelling out numerals zero through one hundred in nontechnical contexts. An alternative rule is spelling out zero through nine. 
"a maximum of five guests"